tests/js/JSONFormatBearTest.py

```diff
@@ -25,6 +26,20 @@
 }"""
 
 
+class ParseExceptionTest(unittest.TestCase):
+
+    def test_parse_err_msg(self):
+        valid_msg = ['Expecting value: line 20 column 30 (char 45)',
+                     'Expecting value: line 1 column 1 ']
+        invalid_msg = ['Expecting property change',
+                       'Expecting line 20 column 30',
+                       'Expecting value: line column']
+        for msg in valid_msg:
+            self.assertTrue(JSONFormatBear.parse_err_msg(msg))
+        for msg in invalid_msg:
+            self.assertFalse(JSONFormatBear.parse_err_msg(msg))
```

Comment:
For tests, I can assert group(1), group(2), group(3) for the expected output. Would that be sufficient?
And by inline, did you mean something like this?
```
def my_function():
    def other_function():
        pass
```